vllm/prompt_adapter/utils.py

```diff
@@ -0,0 +1,91 @@
+import os
+from typing import Optional
+
+import torch
+from huggingface_hub import file_exists, hf_hub_download
+from huggingface_hub.utils import EntryNotFoundError
+from safetensors.torch import load_file as safe_load_file
+
```

Comment:
@prashantgupta24 it would be good to add a comment here and GH permalink to the place in the peft repo from which this method was copied.

Reply:
Done